### PR TITLE
FFmpegUtils - Polish Javadoc

### PR DESCRIPTION
**Pull request by stonio**

### CODE REVIEW

File: src/main/java/net/bramp/ffmpeg/FFmpegUtils.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 /**
  * Helper class with commonly used methods
  */
-public final class FFmpegUtils {
+public abstract class FFmpegUtils {
```
Comment (bramp) [inline src/main/java/net/bramp/ffmpeg/FFmpegUtils.java]: I would expect anyone to extend this class (which is what abstract implies). Instead I used a final class, with private constructor. http://stackoverflow.com/questions/25223553/how-can-i-create-an-utility-class

Comment (bramp) [inline src/main/java/net/bramp/ffmpeg/FFmpegUtils.java]: typo: I wouldn't expect anyone to extend this class (which is what abstract implies). Instead I used a final class, with private constructor.

Comment (stonio) [inline src/main/java/net/bramp/ffmpeg/FFmpegUtils.java]: You're right.

**Comment by bramp** [comment]
There were other unrelated changed in this that were good. Feel free to reopen and rename the problem.

File: src/main/java/net/bramp/ffmpeg/FFmpegUtils.java (hunk 1 above)
Comment (bramp) [inline src/main/java/net/bramp/ffmpeg/FFmpegUtils.java]: If you can change this back to `public final class FFmpegUtils` I can accept the pull request.

**Pull request merged**